docker-compose.yml

```diff
@@ -1,9 +1,26 @@
 version: '3'
 services:
+  dev:
+    build:
+      context: .
+      dockerfile: cmd/proxy/Dockerfile
+    environment:
+      - ATHENS_MONGO_STORAGE_URL=mongodb://127.0.0.1:27017
+      - TIMEOUT=2
+      # - ATHENS_STORAGE_TYPE=mongo
+      - ATHENS_STORAGE_TYPE=memory
+    ports:
+      - 3000:3000
+    depends_on:
+      - mongo
+    networks:
+      - mongo
```

Comment:
do we need a mongo network? docker-compose creates a network by default making all the services reachable

Reply:
@marwan-at-work right you are! I'm gonna push this change shortly